oxli/build_graph.py

```diff
@@ -80,6 +80,7 @@ def main(args):
         check_input_files(fname, args.force)
 
     graphsize = calculate_graphsize(args, 'nodegraph')
+    space_needed = args.n_tables * graphsize / 8
```

Comment:
Oops, there were some changes that were accidentally left out of the last commit. Buuuuuuut...there are still some other issues to fix!